variants/lionbits3/pins_arduino.h

```diff
@@ -0,0 +1,109 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS 40
+#define NUM_ANALOG_INPUTS 16
+
+#define analogInputToDigitalPin(p) (((p) < 20) ? (analogChannelToDigitalPin(p)) : -1)
+#define digitalPinToInterrupt(p) (((p) < 40) ? (p) : -1)
+#define digitalPinHasPWM(p) (p < 34)
+
+static const uint8_t LED_BUILTIN = 0;   // GPIO0, 
+static const uint8_t SWITCH_A = 46;     // GPIO46, 
+static const uint8_t SWITCH_B = 47;     // GPIO47, 
+//Wifi and Bluetooth LEDs
+static const uint8_t WIFI_LED = 38;
+static const uint8_t BT_LED = 37;
+
+
+static const uint8_t TX = 1;
+static const uint8_t RX = 3;
+//-------------------------------------------------------------------
+static const uint8_t U1RX = 9;  //I/O  U1RX     GPIO9
+static const uint8_t U1TX = 10; //I/O  U1TX     GPIO10
+/* LionBits3 pin setup */
+static const uint8_t D0 = 3;    //RX,           GPIO3,                      MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
```

Comment:
I don't get those comments about each pin, as most of the stuff is same for each pin.
For example all the U0xxx_xx ,U1xxx_xx, U2xxx_xx, MCPWM and LEDPWM. These can be on any pins so no need to comment it there.

Please remove those and just leave the good ones, as ADC pin, TOUCH